modules/examples/src/main/java/CreateDbAndDoc.java

```diff
@@ -14,10 +14,10 @@
 import com.ibm.cloud.cloudant.v1.Cloudant;
 import com.ibm.cloud.cloudant.v1.model.Document;
 import com.ibm.cloud.cloudant.v1.model.DocumentResult;
-import com.ibm.cloud.cloudant.v1.model.GetDocumentOptions;
 import com.ibm.cloud.cloudant.v1.model.Ok;
 import com.ibm.cloud.cloudant.v1.model.PostDocumentOptions;
 import com.ibm.cloud.cloudant.v1.model.PutDatabaseOptions;
+// import com.ibm.cloud.cloudant.v1.model.PutDocumentOptions;
```

Comment:
Do we need to add a comment here?
```suggestion
// Uncomment import below if using "putDocument" method
// import com.ibm.cloud.cloudant.v1.model.PutDocumentOptions;
```

Reply:
Done in [61952cb](https://github.com/IBM/cloudant-java-sdk/pull/247/commits/61952cbe80cae1e41a0f5f2d524a668e3c1a6ea1)